### PR TITLE
Post DroidKaigi logo image to "About This App" screen #177

### LINKED ISSUE #177: Post DroidKaigi logo image to "About This App" screen
## Kind (Required)

- Improvement

 ## Overview (Required)

- Post DroidKaigi logo image to "About This App" screen and fix TODO comment

 ## Links	
- Figma: https://www.figma.com/file/IFlrbfmBSdYvUz7VmSzfLV/DroidKaigi_2021_official_app?node-id=0%3A1
スクリーンショット 2021-02-28 21 09 50

- TODO code: https://github.com/DroidKaigi/conference-app-2021/blob/19576a0af53bc1cbf9a5aa0a8e3895de7112dd80/uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/core/AboutThisApp.kt#L50-L51

### PR DESCRIPTION
**Pull request by mightyfrog**
## Issue
- close #177

## Overview (Required)
- Replace a placeholder text with DroidKaig logo image. Add a vector drawable "logo.xml" (name is taken from Figma layer name).

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by mhidaka** [comment]
👀 @mightyfrog Great! we'll review in the morning. please wait. Moreover If you have time, put screenshots (before and after ) on the p-r. Thanks you again for your quick approach.

File: uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/AboutThisApp.kt [hunk 1]
```diff
@@ -47,8 +50,9 @@ fun WhatIsDroidKaigiComponent() {
             style = typography.h6
         )
         Spacer(modifier = Modifier.height(24.dp))
-        // TODO: Replace DroidKaigi Logo
-        Text("ToDo: Replace me to Logo image")
+        Image(
+            painter = painterResource(R.drawable.logo), contentDescription = "DroidKaigi Logo"
```
Comment (takahirom) [inline uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/AboutThisApp.kt:54]: Apparently, it's better to break the parameters, but it's not a big problem and doesn't need to be fixed.
`break each parameter declaration onto its own line`
https://developer.android.com/kotlin/style-guide?hl=en#functions

File: uicomponent-compose/other/src/main/res/drawable/logo.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (takahirom) [inline uicomponent-compose/other/src/main/res/drawable/logo.xml:1]: 👍

**Review by takahirom — APPROVED** [review]
Thank you! LGTM!

File: uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/AboutThisApp.kt (hunk 1 above)
Comment (mightyfrog) [reply uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/AboutThisApp.kt:54]: That's the result of applying spotless.

Comment (takahirom) [reply uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/staff/AboutThisApp.kt:54]: Ah, maybe the spotless option isn't enough, thank you for the information! 🙇

**Pull request merged**